pxtsim/embed.ts

```diff
@@ -100,7 +100,8 @@ namespace pxsim {
         subtype: "steploaded";
         data: {[index: string]: number };
         showCategories?: boolean;
-        location?: string;
+        headercontent: string;
+        content: string;
```

Comment:
The rendering goes on in the iframe and the main frame just displays the html. what's your proposition?
Bear in mind we're trying not to include the runner in the main editor.